### ISSUE TITLE
Panels under a repeated row are grayed out and don't react to time range changes

### ISSUE THREAD

**Issue by Irrational-NX** [grafana/grafana#46108]
**What happened**:
Updated to grafana 8.4.2 and panels that are repeated using a variable do not update when changing time range. Sometimes, they are gray and have "undefined" leggends.
![image](https://user-images.githubusercontent.com/32514952/156407962-cc351107-a147-4db7-9bcd-bfc29217dff4.png)

**What you expected to happen**:
After update, repeated rows and panels should display and react to time range change normaly.

**How to reproduce it (as minimally and precisely as possible)**:
Have panels under a repeated row and try to change the time range.

**Anything else we need to know?**:
The panels under the FIRST repeated row don't seem to have any problems.
Weirdly, if I "close" then re-open a row, the new time range is applicated and the panel retrieve it's original colors.
I do not have any issues with grafana 8.3.7

**Environment**:
- Grafana version: 8.4.2
- Data source type & version: Prometheus
- OS Grafana is installed on: Official Docker image with Ubuntu
- User OS & Browser: Windows and chromium
- Grafana plugins: -
- Others: -

**Comment by ashharrison90** [comment]
hey @Irrational-NX 👋 

thanks for raising this! it doesn't look like it's anything specific to repeating, as that seems to work fine for me and updates all panels when changing time range:
![image](https://user-images.githubusercontent.com/20999846/156548427-36c455a7-d8bb-45f3-aac9-5aebe69fbef2.png)

the `undefined` in the panel axes looks suspicious to me. are you able to share your dashboard json? (`share` -> `export` -> toggle `export for sharing externally`)

**Comment by Irrational-NX** [comment]
Hey @ashharrison90, thank you for looking at this issue! 😃 

Here's the dashboard (compressed into gz because GitHub doesn't want me to upload JSON...) : 
[GPU-dashboard-export.json.gz](https://github.com/grafana/grafana/files/8177522/GPU-dashboard-export.json.gz)
There's a little bit of french in the labels, details, etc... But it should not trouble you.

Let me know if you need more infos.

**Comment by s1sfa** [comment]
I get the same/similar problem also since upgrading to 8.4.2. Sometimes greyed out, and error. Most times, the graph just never changes when variables or time changes. I made a video to show the problem using time change how some panels don't update, even after clicking the grafana refresh button.  [Video of Issue](https://cloudup.com/cqJI64Co8gG)
It doesn't seem to happen on all dashboards with rows.

Upgraded to 8.4.3 and the issue still exists.
I have the problem in Chrome, Firefox, Safari
[Dashboard JSON](https://github.com/grafana/grafana/files/8202538/grafana.json.dashboard.txt)


When my panels are greyed out. If I try to click them, then they show an error. I captured the error in the browser javascript console:
`react-dom.production.min.js:216 TypeError: s.display is not a function
    at h (TooltipPlugin.tsx:233:31)
    at rs (react-dom.production.min.js:157:137)
    at Fs (react-dom.production.min.js:180:154)
    at Wi (react-dom.production.min.js:269:343)
    at Sc (react-dom.production.min.js:250:347)
    at kc (react-dom.production.min.js:250:278)
    at wc (react-dom.production.min.js:250:138)
    at hc (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
si	@	react-dom.production.min.js:216
a.componentDidCatch.n.callback	@	react-dom.production.min.js:217
da	@	react-dom.production.min.js:131
fi	@	react-dom.production.min.js:220
Tc	@	react-dom.production.min.js:259
t.unstable_runWithPriority	@	scheduler.production.min.js:18
Ho	@	react-dom.production.min.js:122
Cc	@	react-dom.production.min.js:252
hc	@	react-dom.production.min.js:243
(anonymous)	@	react-dom.production.min.js:123
t.unstable_runWithPriority	@	scheduler.production.min.js:18
Ho	@	react-dom.production.min.js:122
Vo	@	react-dom.production.min.js:123
Zo	@	react-dom.production.min.js:122
lc	@	react-dom.production.min.js:237
Os	@	react-dom.production.min.js:170
(anonymous)	@	TooltipPlugin.tsx:134
(anonymous)	@	uPlot.esm.js:5186
ro	@	uPlot.esm.js:5185
Ur	@	uPlot.esm.js:4577
Wr	@	uPlot.esm.js:4735
Gr	@	uPlot.esm.js:4933`
